fractal_server/app/api/v1/task.py

```diff
@@ -51,27 +47,42 @@ def get_task(
     return task
 
 
-@router.post("/", response_model=TaskRead, status_code=status.HTTP_201_CREATED)
-async def create_task(
-    task: TaskCreate,
+async def create_task_headless(
+    task_list: List[TaskCreate],
+    db: AsyncSession,
+    global_task: bool = True,
+) -> List[Task]:
+    task_db_list = [Task.from_orm(t) for t in task_list]
+    db.add_all(task_db_list)
+    await db.commit()
+    await asyncio.gather(*[db.refresh(t) for t in task_db_list])
+    return task_db_list
+
+
+@router.post(
+    "/pip/",
+    response_model=List[TaskRead],
+    status_code=status.HTTP_201_CREATED,
+)
+async def create_task_pip(
+    task_collect: TaskCollectPip,
+    collection_type: str = "pip",
+    global_task: bool = True,
     user: User = Depends(current_active_user),
```

Comment:
Possibly trivial question: what does it mean to have `user` as an argument, if it is not used in the endpoint function?

Reply:
My best guess: later on, we'll want to associate some tasks to projects, and projects may be access-limited to certain users. Is this the reason?